GirafRest/Controllers/ActivityController.cs

```diff
@@ -37,13 +37,14 @@ public ActivityController(IGirafService giraf, ILoggerFactory loggerFactory, IAu
         /// <param name="weekplanName">name of the weekplan that you want to add the activity on.</param>
         /// <param name="weekYear">year of the weekplan that you want to add the activity on.</param>
         /// <param name="weekNumber">week number of the weekplan that you want to add the activity on.</param>
-        /// <param name="weekDay">day of the week that you want to add the activity on.</param>
+        /// <param name="weekDayNmb">day of the week that you want to add the activity on (Monday=1, Sunday=7).</param>
         /// <returns>Returns <see cref="ActivityDTO"/> for the requested activity on success else MissingProperties, 
         /// UserNotFound, NotAuthorized, WeekNotFound or InvalidDay.</returns>
-        [HttpPost("{userId}/{weekplanName}/{weekYear}/{weekNumber}/{weekDay}")]
+        [HttpPost("{userId}/{weekplanName}/{weekYear}/{weekNumber}/{weekDayNmb}")]
```

Comment:
Has the endpoint changed?

Reply:
If I understand your question correctly: The endpoint has changed from taking a enum Day parameter named weekDay to indstead taking an int parameter named weekDayNmb, and also typecasting the int to a enum Day is added

Reply:
Okay, so the URL used for the endpoint has not changed?

Reply:
No, only the parameter name and type.
Before the url could be 
"..v2/ActivityController/1/weekplan/2019/1/Monday"
now the corresponding url will be
"":..v2/ActivityController/1/weekplan/2019/1/1"